fe/fe-core/src/main/java/com/starrocks/catalog/EsTable.java

```diff
@@ -261,6 +261,11 @@ private void validate(Map<String, String> properties) throws DdlException {
                 sslEnabled = false;
             }
         }
+        Column idColumn = getColumn("_id");
+        if (idColumn != null && !(idColumn.getPrimitiveType() == PrimitiveType.VARCHAR
+                || idColumn.getPrimitiveType() == PrimitiveType.CHAR)) {
```

Comment:
Is the code indent  correct ?

Reply:
I think maybe correct? it can pass the fe-checkstyle